engine/src/main/java/org/hibernate/search/backend/impl/lucene/WorkspaceHolder.java

```diff
@@ -29,7 +29,7 @@
  * @author Emmanuel Bernard
  * @author Sanne Grinovero
  */
-public class LuceneBackendQueueProcessor implements BackendQueueProcessor {
+public class WorkspaceHolder {
```

Comment:
This might even be pulled into `DirectoryBasedIndexManager`. I left it separately for now to keep the diff a bit nicer.